forge-gui/res/cardsfolder/upcoming/phantasmal_extraction.txt

```diff
@@ -0,0 +1,12 @@
+Name:Phantasmal Extraction
+ManaCost:1 B
+Types:Sorcery
+S:Mode$ ReduceCost | ValidCard$ Card.Self | Type$ Spell | Amount$ X | EffectZone$ All | Description$ This spell costs {1} less to cast if you weren't the starting player.
+SVar:X:Count$StartingPlayer.0.1
+A:SP$ Reveal | ValidTgts$ Opponent | RevealAllValid$ Card.nonLand+TargetedPlayerOwn+cmcLE4| SubAbility$ DBChoice | RememberRevealed$ True | SpellDescription$ Target opponent reveals each nonland card in their hand with mana value 4 or less. Choose a card revealed this way or a card in their graveyard. Exile that card.
+SVar:DBChoice:DB$ GenericChoice | Choices$ ChooseHand,ChooseGraveyard
+SVar:ChooseGraveyard:DB$ ChangeZone | Origin$ Graveyard | Destination$ Exile | Chooser$ You | ChangeType$ Card.TargetedPlayerOwn | IsCurse$ True | Mandatory$ True | SubAbility$ DBCleanup | SpellDescription$ Exile a card from their graveyard
+SVar:ChooseHand:DB$ ChooseCard | ChoiceZone$ Hand | Choices$ Card.IsRemembered | SubAbility$ DBChangeZone | SpellDescription$ Choose one of the cards from that player's hand revealed this way
```

Comment:
This UI seems a bit awkward... may be better to do a Pump line to remember everything in target player's graveyard, then a single ChooseCardEffect with ChoiceZone$ Hand,Graveyard | Choices$ Card.IsRemembered ?

Reply:
![image](https://github.com/Card-Forge/forge/assets/67333662/fc2a110d-47a3-4ad9-babc-b79ef15bb7d9)
thanks